src/App.tsx

```diff
@@ -1,6 +1,32 @@
+import Header from "./components/sections/Header";
+
+interface BEMProps {
+  block: string;
+  element?: string;
+  modifiers?: string[];
+}
+
+const generateBEMClassName = ({ block, element, modifiers }: BEMProps): string => {
+  let className = block;
+
+  if (element) {
+    className += `__${element}`;
+  }
+
+  if (modifiers && modifiers.length > 0) {
+    className += modifiers.map((modifier) => `${block}--${modifier}`).join('');
+  }
+
+  return className
+}
+
 const App = () => {
+
+  const headerClass = generateBEMClassName( { block: 'csApp', element: 'Header' });
   return ( 
-    <div className="csApp"></div>
+    <div className="csApp">
```

Comment:
I don't like this name csApp.  Why not simply appWrapper or something like that?

Reply:
Why you don't like name csApp?

Reply:
What about csAppWrapper?

Reply:
it's because noone will probably know what cs stands for. appWrapper should be enough 👍